data/json/items/tools.json

```diff
@@ -3414,7 +3414,15 @@
     "ammo": "NULL",
     "revert_to": "null",
     "category": "weapons",
-    "use_action": "MOLOTOV"
+    "use_action": {
+        "type": "transform",
+        "target": "MOLOTOV_LIT",
+        "target_charges": 1,
+        "active": true,
+        "need_fire": 1,
+        "need_fire_msg": "You need a source of fire!",
+        "msg": "You light the molotov cocktail"
```

Comment:
Use the message from replaced function: ```You light the Molotov cocktail.``` (your message missed uppercase 'M' and period at the end of the sentence).